### ISSUE TITLE
Idea for overall building process of the programming language and name ideas...

### ISSUE THREAD

**Issue by Xtqm** [Kashina69/RawScript#1]
Hi there @Kashina69! I was just checking out your repo here and really liked the idea you have, however I notice that the process in which you're using to build your programming language isn't the most suitable as I myself have tried it(it did not end well). Sense you're using Python I think you should use the PLY library as it isn't just built for building programming languages, but it also is quite easy to learn once you get the hang of it. If you have any questions just mention me in an issue like this one and I'll be glad to assist you in building this programming language. 

PS: 
Some name ideas for the programming language could be: 
Enpy
Pytonic
In the end, the choice is yours, happy coding!

**Comment by Kashina69** [comment]
@TENTHER101 broo thank for commenting this you can't even immagine how much happy i am seeing your comment really really gladd thank for the advice thoo and i will work on that too hwo you found my repoo tell me and also i was like this will be some another side project of mine which will be just half done but your comment motivates me to complete it 
actually i knew that python is not the best thing for this like slow lang making another slow lang but i wanted to make a mvp to get my foot and faster dev speed i am a js dev but js would not be the best for fast dev time my actually plan is to convert it after making mvp to rust or golang caus i want to learn those languages and they are fast tooo really like those two languages even i haven't tryed them and i was thinking of something like regex but it was too much for mee 
and what was your programming language name btw 
i am planning to give it the name of RawScript suits its scripting style and i am a js dev and also its Raw like english 
i am done some changes in this code and pushed it check that out too 
and thxx again

**Comment by Xtqm** [comment]
Yes, of course! I am really trying to become part of this community so I want to start bit by bit. I was just exploring for new programming languages being developed due to my fascination for them and so I searched up "custom programming language language:python" and then filtered it to view the most recently updated. Yours was one of the first ones there and so I checked it out. My bad, didn't mean to say it isn't good for building programming languages, I meant it is good. Sorry for the misunderstanding. Even though Python is slow, it can still be extremely useful and powerful. Rust and Go would be good choices, however they are much more complicated than Python, so do take that into consideration. 
Oh, my programming language is named TEPL(short for textual educational programming language), you can check it out here: https://github.com/TENTHER101/TEPL. It's still in development and is not useful right now, however you can check out past versions by going through the history of commits I've made and find a working one to use. 
RawScript sounds like a good name, just double check it doesn't already exist! 
Ok, will do!
Happy coding!
